swarm/network/hive.go

```diff
@@ -487,13 +517,14 @@ func (h *Hive) getRetriesFromDuration(timeAgo time.Duration) int {
 }
 
 func (h *Hive) isTimeForRetry(d *Peer) bool {
+	//	debug.PrintStack()
```

Comment:
this comment is for `getRetriesFromDuration` ON LINE 511
what is this?

```
 div += (150000 - rand.Int63n(300000)) * div / 1000000
```

Reply:
not sure i understand the question. anyway removed by now